apps/apps_container.h

```diff
@@ -24,6 +24,9 @@
 
 class AppsContainer : public Container, ExamPopUpControllerDelegate, Ion::StorageDelegate {
 public:
+  static AppsContainer * sharedAppsContainer() {
+    return static_cast<AppsContainer *>(Container::sharedContainer());
+  }
```

Comment:
Remove because AppsContainerStorage::sharedContainer() can be used instead.
(don't forget to change #include "apps_container.h" to #include "apps_container_storage.h")

Reply:
Container::sharedContainer() and AppsContainerStorage::sharedContainer() have been removed. Only AppsContainer::sharedAppsContainer() is left.